_guide/_pages/interviews/core-values.md

```diff
@@ -54,108 +54,61 @@ Begin by introducing yourself by saying this or something similar to it:
-
-- [What is your comfort level with peer-reviewed work?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
-    - If applicable: Have you proactively sought additional training and/or peer review?
-
-- [How would you deal with a bug in an open source project?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
 
 ### Diversity, equity, and inclusion
 
 Diversity, equity, and inclusion are core values for 18F and the engineering chapter. These questions are meant to shed light on the candidate's experience and thoughts.
 
-- "At 18F we strive to be as representative of the populace as possible and work hard to ensure that everyone understands this notion and is treated fairly and respectfully. [What do diversity, equity, and inclusion mean to you?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.5kwvj8b9hvhd)"
+- "[Tell me about a time you’ve worked with or for communities different than your own. What did you learn?](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.u30z4bws4grp)"
+    - What do diversity and inclusion mean to you?
+    - How do you take equity into consideration in your software practice?
 
 ### Overall experience and outlook
 
 Begin closing the interview by asking general questions to ensure we develop a more well-rounded picture of the candidate. These questions aren’t geared toward a specific job or work methodology. Instead, they’re meant to shed light into the candidate’s overall experience and outlook.
 
 Ask one or more of the following:
 
-- [Tell me about a time that something at work didn’t go well.](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.6e8l8bhmhbi3)
-- [Tell me about a time you worked with or for communities different than your own.](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.6e8l8bhmhbi3)
-- [Tell me about what you’ve done that you’re most proud of.](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.6e8l8bhmhbi3)
-
-### Conclusion
-
-Wrap up the interview by tying things back to why the candidate is looking to join 18F. Hopefully the answers and discussions around the specific questions provide a modicum of context around the candidate’s motivations and desires, in which it will be a lot easier to determine if their personal and professional goals align with 18F’s needs and mission.
-
-Ask one or more of the following:
+- [At 18F, we sometimes find ourselves needing to create a strategic direction from  conflicting expectations. Tell me about a time you had to reset expectations with a product owner or customer and how you went about it.](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.2i6wokvb700e)
```

Comment:
I’m worried that this might not translate well to candidates who haven’t worked in a consulting environment before coming to 18F (myself!). 

I think we’d be able to achieve the same question without the domain-specific knowledge requirement: 

> At 18F, we sometimes find ourselves needing to create a strategic direction from conflicting expectations. Tell me about a time you had to reset expectations others had for your work, and how you went about it.